zzz_generated/app-common/app-common/zap-generated/cluster-objects.h

```diff
@@ -34150,8 +34170,8 @@ struct Type
     static constexpr CommandId GetCommandId() { return Commands::TestEnumsResponse::Id; }
     static constexpr ClusterId GetClusterId() { return Clusters::TestCluster::Id; }
 
-    chip::VendorId arg1;
-    SimpleEnum arg2;
+    chip::VendorId arg1 = static_cast<chip::VendorId>(0);
+    SimpleEnum arg2     = static_cast<SimpleEnum>(0);
```

Comment:
What about enums that don't have a valid enum defined for 0?

Reply:
Per spec, the type default for enumerations is MS.

I don't know that there's any way to win here, really, short of moving to getting the default value from the XML (i.e. using the actual spec-defined default).  Which we should ideally do, of course.